src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -37,6 +37,8 @@ export default function CIDetails({ isJobView }: { isJobView?: boolean }) {
     const [triggerHistory, setTriggerHistory] = useState<Map<number, History>>(new Map())
     const [fullScreenView, setFullScreenView] = useState<boolean>(false)
     const [hasMoreLoading, setHasMoreLoading] = useState<boolean>(false)
+    const [aretagsEditable, setAreTagsEditable] = useState<boolean>(false)
```

Comment:
we can prevent these 2 states and use it directly from triggerHistoryResult

Reply:
done